Resources/Prototypes/_Lust/Arrows/Arrows_implants.yml

```
@@ -0,0 +1,200 @@
+    - type: Implanter
+      implant: SmiteImplant
+
+#ИМПЛАНТ ХРИП
+- type: entity
+  parent: DeathRattleImplant
+  id: DeathRattleImplantMT
+  name: Death rattle implanter
+  description: This implant will notify the Arrows radio channel if the user is in critical condition or dies.
+  noSpawn: true
+  components:
+  - type: Rattle
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)